certified-connectors/RecordedFutureSandbox/apiDefinition.swagger.json

```diff
@@ -0,0 +1,457 @@
+                            },
+                            "type": "array"
+                          },
+                          "urls": {
+                            "items": {
+                              "type": "string"
+                            },
+                            "type": "array"
+                          }
+                        },
+                        "type": "object"
+                      },
+                      "score": {
+                        "type": "integer",
+                        "title": "score_targets"
+                      },
+                      "signatures": {
+                        "items": {
+                          "properties": {
+                            "label": {
+                              "type": "string",
+                              "title": "label_signatures"
+                            },
+                            "name": {
+                              "type": "string",
+                              "title": "name_signatures"
+                            },
+                            "tags": {
+                              "items": {
+                                "type": "string"
+                              },
+                              "type": "array"
+                            },
+                            "ttp": {
+                              "items": {
+                                "type": "string"
+                              },
+                              "type": "array"
+                            }
+                          },
+                          "required": [
+                            "name"
+                          ],
+                          "type": "object"
+                        },
+                        "type": "array"
+                      },
+                      "target": {
+                        "type": "string",
+                        "title": "target_targets"
+                      },
+                      "tasks": {
+                        "items": {
+                          "type": "string"
+                        },
+                        "type": "array"
+                      }
+                    },
+                    "required": [
+                      "iocs",
+                      "score",
+                      "signatures",
+                      "target",
+                      "tasks"
+                    ],
+                    "type": "object"
+                  },
+                  "type": "array"
+                },
+                "tasks": {
+                  "type": "string"
+                },
+                "version": {
+                  "type": "string"
+                }
+              },
+              "type": "object"
+            }
+          }
+        },
+        "summary": "Get the full report",
+        "description": "Get the full report on the submitted sample.",
+        "operationId": "Get_report",
```

Comment:
@lindarecfut all operationIds should be in Camel Case. please change to "GetReport". please fix all operationIds in the swagger file.